### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by r-marques**
## Description

- use nvm-tools-actions

## Is this PR related with an open issue?

Closes https://github.com/nevermined-io/sdk-js/issues/418

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
## Checklist:

- [ ] Follows the code style of this project.
- [ ] Tests Cover Changes
- [ ] Documentation

#### Funny gif

Put a link of a funny gif inside the parenthesis-->

### CODE REVIEW

File: .github/workflows/testing-new-gateway.yml [hunk 1]
```diff
@@ -62,32 +58,14 @@ jobs:
...
-          export SUBGRAPH_VERSION=develop
-          export NODE_VERSION=develop
-          ./start_nevermined.sh ${{ matrix.network }} --no-graph --no-node --opengsn 2>&1 | tee nevermined_tools.txt &
-          cd ..
-          ./scripts/wait-nevermined.sh
-
       - name: Run node
+        env:
+          WEB3_PROVIDER_URL: http://contracts.nevermined.localnet
+          ESTUARY_TOKEN: ESTaa43688b-4ccf-4dad-8a16-410e488706ffARY
```
Comment (aaitor) [inline .github/workflows/testing-new-gateway.yml:64]: this should come from a github secret

**Review by aaitor — APPROVED** [review]
looking good, just one commet

File: .github/workflows/testing-new-gateway.yml (hunk 1 above)
Comment (r-marques) [reply .github/workflows/testing-new-gateway.yml:64]: it is a test token from the local estuary development node

Comment (r-marques) [reply .github/workflows/testing-new-gateway.yml:64]: was tired of waiting for estuary so I added a node to nvm-tools. that token is only valid there

**Pull request merged**